File: new_course/pattern/exm-4.js
//     *
//    ***
//   *****
//  *******
// *********

const rows = 5;

for (let i = 1; i <= rows; i++) {
    let stars = '';
    // for printing spaces
    for (let j = 1; j <= rows - i; j++) {
        stars += ' ';
    }
    // for printing stars
    for (let k = 1; k <= (i * 2) - 1; k++) {
        stars += '*';
    }
    console.log(stars);
}

File: new_course/pattern/exm-4.cpp
//     *
//    ***
//   *****
//  *******
// *********

#include <iostream>
using namespace std;

int main()
{
    int rows;
    cout << "Enter the number of rows: ";
    cin >> rows;

    // Outer loop for the number of rows
    for (int i = 1; i <= rows; ++i)
    {
        // Inner loop for printing spaces
        for (int j = 1; j <= rows - i; ++j)
        {
            cout << " ";
        }
        // Inner loop for printing '*' in each row
        for (int k = 1; k <= 2 * i - 1; ++k)
        {
            cout << "*";
        }
        // Move to the next line after printing all '*' in the row
        cout << endl;
    }

    return 0;
}
